Embed WebVI into Content/Readme.md

```diff
@@ -86,13 +104,13 @@ To provide feedback to the 375px bounds of `.content-item`, CSS has been added t
 ```
 Once the content is placed within this box and the WebVI is built we have the correct CSS needed to achieve the same layout within a `.container`.
 
-![Render black box in LabVIEW](docs/box-in-lv.PNG)
+<img src="box-in-lv.PNG" alt="Render black box in LabVIEW text" width="50%">
 
 ### Build Process
 Open `WebApp.gcomp` go to the **Document** tab and click **Build**. This produces the HTML (`Main.html`)of the front panel, the CSS defining the controls' absolute layout, and the compiled WebVI block diagram (`Main.via.txt`).
 
 ## Coping HTML Custom Elements, Styles, and References
-This is the most brittle part of the process. If a controls is removed, added, or replaced this copy/paste process must occur again to incorporate them into the final page. In future revisions of LabVIEW and this example this process could be better automated. Changes to the block diagram are less brittle since we can refer to the .via.txt using a relative path that does not change when the Web application is rebuilt.
+This is the most brittle part of the process. If a controls is removed, added, or replaced this copy/paste process must occur again to incorporate them into the final page. In future revisions of LabVIEW NXG and this example this process could be better automated. Changes to the block diagram are less brittle since we can refer to the .via.txt using a relative path that does not change when the Web application is rebuilt.
```

Comment:
You're right, while it could be better I don't really know how or why we would do something like this in the near term. I'll be removing this line.